app.py

```diff
@@ -66,29 +75,50 @@ def snap_details(snap_name):
     some of the data through to the snap-details.html template,
     with appropriate sanitation.
     """
+    today = datetime.date.today()
+    month_ago = today - relativedelta.relativedelta(months=1)
```

Comment:
What about using UTC timestamps ? ```today = datetime.datime.utcnow().date()```
Also, I am not sure how you intend to display 1 month of installed_base data. Perhaps the delta between day 1 & day 30 ? The simplest thing the occurs to me, would be to show installed_base 'yesterday'.

Reply:
Good point, I'll use `utcnow`.